recipes-kernel/linux/files/bcm2712-raspberrypi5-domd.dts

```diff
@@ -63,6 +63,59 @@
 				      0x00 0x9e 0x04 0x00 0x9f 0x04 0x00 0xa0 0x04>;
 		};
 
+		mip1: msi-controller@131000 {
+			compatible = "brcm,bcm2712-mip-intc";
+			reg = <0x10 0x00131000  0x0 0xc0>;
+			msi-controller;
+			interrupt-controller;
+			#interrupt-cells = <2>;
+			brcm,msi-base-spi = <247>;
+			/* Actually 20 total, but the others are
+			 * both sparse and non-consecutive */
+			brcm,msi-num-spis = <8>;
+			brcm,msi-offset = <8>;
+			brcm,msi-pci-addr = <0xff 0xffffe000>;
+			xen,path = "/axi/msi-controller@131000";
+			xen,force-assign-without-iommu;
+			xen,reg = <0x10 0x00131000 0x0 0x1000 0x10 0x00131000>;
+		};
+
+		pciex1: pcie@1000110000 {
```

Comment:
Why pcie has the following name: pcie@1000110000 and alias pciex1?
While original device is pcie@110000 and has both aliases: pciex1 and pcie1? 
Same for pciex: pcie@1000120000 
Original dts has pciex4: pcie@120000
@GrygiriiS

Reply:
No. No. Please think and check before doing changes !. Original DT has **`"ranges"`** properties, but target, domain DT does not. Therefor target domain passthrough nodes **MUST** be defined using full 64-bit addresses.

I have pretty bad feeling that with this change it will not build (or run).